plugins/@grouparoo/airtable/__tests__/with-meta-access/export-records.ts

```diff
@@ -0,0 +1,6 @@
+import { testExportRecords } from "../utils/export-records";
+import { helper } from "@grouparoo/spec-helper";
+import { getUpdater } from "../utils/nockHelper";
+
+helper.useNock(__filename, getUpdater("META"));
+testExportRecords("META");
```

Comment:
Same here - what's this test up to?

Reply:
These both run the same test (from "../utils/export-records") because it should works exactly the same in both.
I decided not to duplicate the test exactly twice because it's the most important one and I didn't want them to diverge.